docs/src/main/sphinx/release/release-357.md

```diff
@@ -0,0 +1,85 @@
+* Add {func}`format_number` function. ({issue}`1878`)
+* Change `row` to `json` cast to produce JSON objects instead of JSON arrays. This behavior can be restored 
+  with the `deprecated.legacy-row-to-json-cast` configuration option. ({issue}`3536`)
+* Print dynamic filters summary in `EXPLAIN ANALYZE`. ({issue}`7874`)
+* Improve performance for queries using `IN` predicate with a short list of constants. ({issue}`7840`)
+* Reduce memory usage for queries with `WINDOW` clause. ({issue}`7947`)
+* Fix incorrect handling of row expressions in `IN` predicate, quantified comparisons and scalar subqueries. Previously,
+  the queries would succeed where they should have failed with a type mismatch error. ({issue}`7797`)
+* Fix failure when preparing a `GRANT` statement that contains quoted SQL keywords. ({issue}`7941`)
+
+## Security
+
+* Enforce materialized view creator security policies when view is fresh. ({issue}`7618`)
+* Do not require configuring truststore for trusted by root CA certficates for Oauth2 or JWK authentication when 
+  secured internal communication is used. ({issue}`7936`)
+
+## Web UI
+
+* Make the UI aware of principal-field (configured with `http-server.authentication.oauth2.principal-field`) when 
+  `web-ui.authentication.type` is set to `oauth2`. ({issue}`7526`)
+
+## JDBC driver
+
+* Cancel Trino query execution when JDBC connection or statement is closed. ({issue}` 7819`)
+             
+## CLI
+
+* Add `clear` command to clear the screen. ({issue}`7632`)
+
+## BigQuery connector
+
+* Fix failures for queries accessing `information_schema.columns` when `case-insensitive-name-matching` is disabled. ({issue}`7830`)
+* Fix query failure when a predicate on a BigQuery `string` column contains a value with a single quote (`'`). ({issue}`7784`)
+
+## ClickHouse connector
+
+* Improve performance of aggregation queries by computing aggregations within ClickHouse. Currently, the following aggregate functions 
+  are eligible for pushdown: `count`,  `min`, `max`, `sum` and `avg`. ({issue}`7434`)
+* Map ClickHouse `UUID` columns as `UUID` type in Trino instead of `VARCHAR`. ({issue}`7097`)
+
+## Elasticsearch connector
+
+* Support decoding `timestamp` columns encoded as strings containing milliseconds since epoch values. ({issue}`7838`)
+* Retry requests with backoff when Elasticsearch is overloaded. ({issue}`7423`)
+
+## Kinesis connector
+
+* Allow configuring refresh interval to get table descriptions from S3 via the `kinesis.table-description-refresh-interval` config property. ({issue}`1609`)
+
+## Kudu connector
+
+* Fix query failures for grouped execution on range partitioned tables. ({issue}`7738`)
+
+## MongoDB connector
+
+* Redact the value of `mongodb.credentials` in the server log. ({issue}`7862`)
+* Add support for dropping columns. ({issue}`7853`)
+
+## MySQL connector
+
+* Add JSON based identifier mapping that can be set with `case-insensitive-name-matching.config-file`. ({issue}`7841`)
```

Comment:
"JSON based identifier mapping" sounds strange. Maybe say
```markdown
Add rule support for identifier mapping. The rules can be configured via the 
`case-insensitive-name-matching.config-file` configuration property.

Reply:
This probably needs to go in a "Other connectors" since it seems to affect all JDBC based connectors including Phoenix, Redshift, MemSQL, Druid, etc.